x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -127,8 +148,29 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     );
 
   describe('Hosts View', function () {
+    let synthtraceApmClient: any;
     before(async () => {
+      const kibanaClient = new ApmSynthtraceKibanaClient({
+        target: getKibanaServerUrl(),
+        logger: createLogger(LogLevel.debug),
+      });
+      const kibanaVersion = await kibanaClient.fetchLatestApmPackageVersion();
+      await kibanaClient.installApmPackage(kibanaVersion);
+      synthtraceApmClient = new ApmSynthtraceEsClient({
```

Comment:
Rather than creating the client inside a test file, you can create the client at FTR Context level and access it with `getService`. This way all the test files can access synthtrace easily.

For reference on how you can do this - https://github.com/elastic/kibana/blob/65291c64932389290cdf3a81017b3cd4f54bd143/x-pack/test/functional/apps/observability_logs_explorer/config.ts

Reply:
I decided not to do that because it causes our [Firefox tests](https://github.com/elastic/kibana/blob/main/x-pack/test/functional/config.firefox.js) due to it not having access to the app config.

See https://github.com/elastic/kibana/issues/175064.  I'm planning to followup creating a shared service.

Reply:
I just saw the the broken types in Log Explorer. It's a good catch, i will push a fix for Log Explorer soon.

Regarding creating it as a shared service, that could be a really good idea but that could also be expensive due to it installing APM Package Integration on initialisation. 

When i extended it to Log Explorer, i found that it was being extended for Infra and other projects and thought of moving it to common but then i was not sure how bad that would impact the performance. But happy to see it getting some more love and being extended to common.

Reply:
Regarding FF test, my understanding was Appex team is slowing deprecating FF tests.

Reply:
Yea, I'm thinking there might need to be a work around for installing the APM integration.  There is some chat happening about it I'll send you.

Seems like the appex team is letting plugin owners decide whether to be in FF or not.  I did talk to Felix and he said it would probably be fine for us to remove it (affects logs ui).  I didn't necessarily want to remove it because of this issue, though, as appex said we probably should not be fetching that integration in a service anyway.